### DISCUSSION TITLE
On docker-compose up Error: Command "sequelize:migrate" not found.

### DISCUSSION THREAD

**Discussion by JordanBettridge** [outline/outline#5419] [Self Hosting] [answered]
After adding the docker-compose.yml file I try docker-compose up it and I get the error
outline_1   | error Command "sequelize:migrate" not found.

I am using the latest version that is on docker, 

Here is a sanitized copy of my docker file
[docker-compose.txt](https://github.com/outline/outline/files/11629997/docker-compose.txt)

**Comment by routerino** [comment, marked as answer]
outline doesn't need to be migrated manually anymore. You just need `command: sh -c "yarn start --env=production-ssl-disabled"`. See https://blog.gurucomputing.com.au/s/blog/doc/installing-outline-tXZO9ehoFV

**Comment by tommoor** [comment]
The error is correct, the command `sequelize:migrate` doesn't exist anywhere in the codebase – perhaps an outdated guide?

**Reply by JordanBettridge** [reply]
I believe it was an outdated guide. Though I am now running into a new issue, I run docker up, and the final log is "Listening on http://localhost:3000" so I now run docker up -d and then I open my nginx proxy manager, I add a new proxy but I get an error 502, I have tried the following proxy settings

scheme is always http
port is always 3000
IP/Forward  I have tried, outline, outline_outline_1, 0.0.0.0, 127.0.0.1, localhost, and one or two others.
I always get an error 502, and sometimes 
{"status":"OK","version":{"major":2,"minor":10,"revision":3}}

Here is my docker file, please note I only added net=host recently to the yaml file.

https://gist.github.com/Jordz2203/753257dfd1e791570e92abc1892f52fd docker.env
https://gist.github.com/Jordz2203/04744f87eaf43676cb97cfd03971b4b4 docker-compose.yml

**Reply by JordanBettridge** [reply]
If need be I can send you a full working docker-compose file but it would contain all the working keys I am using so need to private send
